content/_index.md

```diff
@@ -4,27 +4,66 @@ date: 2019-06-28
 disableToc: false
 ---
 
-# Packit
+{{< columns >}}
 
-Packit provides tooling and automation to integrate upstream open source
-projects into [Fedora Linux](https://getfedora.org) or [CentOS Stream](https://www.centos.org/centos-stream/).
-Packit project is composed of two main components:
+### What is Packit?
 
-1. [packit]({{< ref "cli" >}}), a CLI tool, which you can
+1. a [CLI tool]({{< ref "cli" >}}), which you can
    [install locally](/docs/guide/#have-packit-tooling-installed-locally)
-   and start using right away.
+   and start using right away,
 
-2. [Packit Service]({{< ref "packit-service" >}}), can provide you feedback
-   on how is your project integrated with Fedora Linux or CentOS Stream.
+2. a [service]({{< ref "packit-service" >}}), which provides you feedback
+   on how is your project integrated with [Fedora Linux](https://getfedora.org), [CentOS Stream](https://www.centos.org/centos-stream/)
+   and other distributions.
+   <--->
 
-Feel free to jump into [the guide]({{< ref "guide" >}}) for using packit.
+### Who should use Packit?
 
-### Contact
+Upstream maintainers who would like to automate validation, testing and releasing of code changes
+in downstream distributions.
+{{< /columns >}}
 
-If you encounter a problem while using packit or Packit Service, please open an
-[upstream issue](https://github.com/packit/packit-service/issues/new).
-In case of any other questions, feel free to contact
-[us](https://github.com/orgs/packit/teams/the-packit-team):
+### What key features does Packit offer?
 
-- **#packit** on [Libera.Chat](https://libera.chat/)
-- **hello@packit.dev**
+{{< columns >}}
+
+#### Building RPMs
+
+Packit uses [Copr build system](https://copr.fedorainfracloud.org/) to build RPMs from your code changes,
+so you can validate your package will be correctly built from each pull request or have automation around your Copr
+repository containing builds from your upstream releases/commits.
+<--->
+
+#### Testing
+
+Packit runs your tests in [Testing Farm](https://docs.testing-farm.io/general/0.1/index.html) and can test either the RPM package built in Copr or
+trigger tests directly on different platforms.
+<--->
+
+#### Automating Fedora releases
+
+With Packit, you can easily
+get your upstream releases to the [Fedora dist-git](https://src.fedoraproject.org), automatically submit builds in [Koji](https://koji.fedoraproject.org/koji/), and we are currently
```

Comment:
Do we have this actually somewhere documented (from top view)? I am thinking of link "learn more".

Reply:
The "get your upstream releases" is documented as `propose-downstream` job and `Koji` builds will be available once we merge #403. (We can add the link later on.)

And the whole overview can be seen here #404. (We are discussing [here](https://github.com/packit/packit.dev/pull/404#discussion_r826630151) about reusing/updating this to be the overall guide or creating a new, similar, text that will cover this.

Reply:
I would probably add the link to the whole overview when it will be merged (not sure about linking the specific jobs, since they should be able to reach from the overview and the previous `features` also do not directly link the jobs.